config/admin-functions/create-machine-cert.sh

```diff
@@ -0,0 +1,173 @@
+    local machine_jurisdiction
+    while true; do
+        read -p "${prompt}" machine_jurisdiction
+        if [[ "${machine_jurisdiction}" =~ ^[a-z]{2}\.[a-z-]+$ ]]; then
+            mkdir -p "${VX_CONFIG_ROOT}"
+            echo "${machine_jurisdiction}" > "${VX_CONFIG_ROOT}/machine-jurisdiction"
+            break
+        else
+            echo -e "\e[31m${validation_error_message}\e[0m" >&2
+        fi
+    done
+    echo "${machine_jurisdiction}"
+}
+
+# USB helpers
+
+# Returns the path to the first connected USB stick, if any. Returns an empty string if no USB
+# stick is connected.
+function get_usb_path() {
+    lsblk /dev/disk/by-id/usb*part* --noheadings --output PATH 2> /dev/null | grep / --max-count 1
+}
+
+# Errs if no USB is mounted (or if unmounting fails for some other reason)
+function unmount_usb() {
+    "${VX_METADATA_ROOT}/app-scripts/unmount-usb.sh"
+}
+
+# Never errs, even if no USB is mounted
+function unmount_usb_if_mounted() {
+    "${VX_METADATA_ROOT}/app-scripts/unmount-usb.sh" 2> /dev/null || true
+}
+
+function mount_usb() {
+    local usb_path="${1}"
+    unmount_usb_if_mounted
+    "${VX_METADATA_ROOT}/app-scripts/mount-usb.sh" "${usb_path}"
+}
+
+function mount_usb_if_present() {
+    local usb_path="$(get_usb_path)"
+    if [[ -n "${usb_path}" ]]; then
+        mount_usb "${usb_path}"
+    fi
+}
+
+function wait_for_usb_and_mount_once_present() {
+    while true; do
+        local usb_path="$(get_usb_path)"
+        if [[ -n "${usb_path}" ]]; then
+            mount_usb "${usb_path}"
+            break
+        fi
+        sleep 1
+    done
+}
+
+# OpenSSL helpers
+
+function generate_private_key() {
+    local private_key_path="${1}"
+    local private_key_password="${2}"
+    openssl ecparam -genkey -name prime256v1 -noout | \
+        openssl pkcs8 -topk8 -passout "pass:${private_key_password}" -out "${private_key_path}"
+}
+
+function construct_machine_cert_subject() {
+  local machine_jurisdiction="${1:-}"
+  if [[ -n "${machine_jurisdiction}" ]]; then
+    echo "/C=US/ST=CA/O=VotingWorks/${VX_IANA_ENTERPRISE_OID}.1=${VX_MACHINE_TYPE}/${VX_IANA_ENTERPRISE_OID}.2=${machine_jurisdiction}/"
+  else
+    echo "/C=US/ST=CA/O=VotingWorks/${VX_IANA_ENTERPRISE_OID}.1=${VX_MACHINE_TYPE}/"
+  fi
+}
+
+function create_machine_cert_signing_request() {
+    local machine_private_key_path="${1}"
+    local machine_private_key_password="${2}"
+    local machine_cert_subject="${3}"
+    local cert_signing_request_path="${4}"
+    openssl req -new \
+        -config "${VX_METADATA_ROOT}/vxsuite/libs/auth/certs/openssl.cnf" \
+        -key "${MACHINE_PRIVATE_KEY_PATH}" \
+        -passin "pass:${machine_private_key_password}" \
+        -subj "${machine_cert_subject}" \
+        -out "${cert_signing_request_path}"
+}
+
+# ---------- Script ----------
+
+# TODO: Use the machine's TPM private key (generated by the generate-key.sh script) instead of a
+# new key
```

Comment:
As discussed outside of this PR, an intentional intermediate state until we can figure out how to run relevant openssl commands with a TPM private key